themes/digital.gov/src/scss/new/guides/_guide-landing.scss

```diff
@@ -0,0 +1,60 @@
+@use "uswds-core" as *;
+
+.dg-guides {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding-top: units(2);
+
+  @include at-media("tablet") {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include at-media("desktop") {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.dg-guides-card {
+  border: 2px color("gray-cool-10") solid;
+  border-radius: units(1);
+  display: grid;
```

Comment:
Is this needed? The card is already a single column.

Reply:
Yes, removing breaks the layout.